content/_guides/catalyst.md

```diff
@@ -0,0 +1,169 @@
+conversation on them and their thoughts, instead of on yourself.
+
+Try to appear understanding of their concern rather than dismissive, even if
+you disagree. Your goal is to calm them down, and being oppositional will not
+achieve this.
+
+When requesting someone change their behaviour, instead of ordering them to do
+or not do something, you can also ask them if they think they can. They will
+probably be more receptive and perhaps offer ways you can help them achieve
+it.
+
+### Try to remain calm
+
+People learn by example and often respond in kind. Thus, if you demonstrate
+calmness when initiating a conversation, the conversation is more likely to
+proceed that way.
+
+Likewise, being respectful and professional when dealing with an upset person
+will do a lot of work towards forming a treaty with them.
+
+### Compromising, conceding, and resolutions with no winner
+
+Humility is essential when in leadership positions.
+
+Sometimes, you will need to compromise or concede. Digging in when you make a
+mistake will not improve your community and could lead to distrust.
+
+You can always amend a lenient decision or concession in the future if there
+is actually a need to do so.
+
+## Remember to take breaks to avoid burn-out
+
+When previously rewarding tasks, like helping in channels, becomes tiring and
+less enjoyable, this is a sign of burn-out.
+
+If you manage a space alone, it is easy to feel obligated to spend all your
+energy and time keeping the place in order. This can lead to burn-out, which
+makes it much harder to handle channel conflicts calmly. When that happens,
+your community members feel like they're walking on eggshells. It is also
+easier for bad actors to weaponize you against other community members.
+
+If you are the only channel op, we encourage you to consider giving trusted
+channel users ops as well. Not only does this provide you with the freedom to
+do other things without worrying about the channel, but it also gives you
+another person to talk with about situations that arise.
+
+Find others in your community who are good at talking to people to help you
+manage things. They don't need to be the most knowledgeable person in your
+channel, but they need to be patient, fair, and approachable.
+
+## Appeals processes and accountability
+
+Removing people should ideally not be a permanent thing. People make mistakes
+and have bad days. One mistake or bad day shouldn't be the end of their
+participation in your channel.
+
+Also, permanently excluding users from a channel can lead to them seeking a
+way of evading into the channel, becoming even more problematic.
+
+It's an excellent idea for your channel to have a way for people to appeal a
+ban and negotiate being allowed back. Consider having an "ops" channel; most
+channels use `#<channelname>-ops` for this. Discourage non-ops from lingering
+in the channel, so people who approach you can do so with some privacy. When
```

Comment:
```suggestion
channels use `#<channelname>-ops` for this. Encourage non-ops to leave the channel once
their case has been handled, so people who approach you can do so with some privacy. When
```

I think this would be a more positive way to communicate the same thing.

Reply:
Well, now it's a bit specific to those who even had a case to start with and not also tourists. I'll come up with a compromise.

Reply:
Does the new wording i've committed address your concern?

Reply:
Yes, thank you.